roles/edpm_nova/tasks/configure.yml

```diff
@@ -118,3 +120,33 @@
     mode: '0600'
     owner: nova
     group: nova
+
+- name: get fact for tls_certs_enabled
+  set_fact: tls_certs_enabled="{{ lookup('env','tls_certs_enabled') }}"
+
+- name: copy cert file to {{edpm_nova_certs_dest}}
+  tags:
+    - configure
+    - nova
+  ansible.builtin.copy:
+    src: "{{ edpm_nova_certs_src }}/{{ inventory_hostname }}-tls.crt"
+    dest: "{{edpm_nova_certs_dest}}/tls.crt"
+  when: tls_certs_enabled
+
+- name: copy key file to {{edpm_nova_certs_dest}}
+  tags:
+    - configure
+    - nova
+  ansible.builtin.copy:
+    src: "{{ edpm_nova_certs_src }}/{{ inventory_hostname }}-tls.key"
+    dest: "{{edpm_nova_certs_dest}}/tls.key"
+  when: tls_certs_enabled
+
+- name: copy cacert file to {{edpm_nova_cacerts_dest}}
+  tags:
+    - configure
+    - nova
+  ansible.builtin.copy:
+    src: "{{ edpm_nova_cacerts_src }}/TLSCABundleFile"
+    dest: "{{edpm_nova_cacerts_dest}}/TLSCABundleFile"
+  when: tls_certs_enabled
```

Comment:
since the when condition is the same between these 3 copies
i would consider refactoring this into a loop

the other thing to consider is these files likely need to be owned by the nova user

which is created above.

the nova user on the host ahs the same uid and gid as the nova user in the container.

you can presumable use the same permissions as we are doing for the ssh key above

i.e. add  the following to each copy 
```
mode: '0600'
owner: nova
group: nova
```

Reply:
ack will refactor and add ownership